spec/unit/system_description_spec.rb

```diff
@@ -75,6 +76,37 @@
     end
   end
 
+  context "meta data methods" do
+    let(:date) { "2014-02-07T14:04:45Z" }
+    let(:hostname) { "example.com" }
+    let(:name) { "example" }
+    let(:date_human) { Time.parse(date) }
```

Comment:
Do not use `Time.parse` without zone. Use one of `Time.zone.parse`, `Time.parse.current`, `Time.parse.in_time_zone`, `Time.parse.utc`, `Time.parse.getlocal`, `Time.parse.iso8601`, `Time.parse.jisx0301`, `Time.parse.rfc3339`, `Time.parse.to_i`, `Time.parse.to_f` instead.